GoogleForCreators/web-stories-wp
# Workspace: Fix crash where we have multiple elements with multiple colors selected

## Summary

Fixes the generate color pattern function to handle a selection with multiple values.

<img width="950" alt="Screen Shot 2020-10-07 at 5 07 43 PM" src="https://user-images.githubusercontent.com/1738349/95393402-27d5d400-08c0-11eb-8677-0f7590af44b6.png">


## Relevant Technical Choices

Follows the existing pattern of returning `''` to denote multiple values to the panel.

## To-do

None

## User-facing changes

The editor should not crash anymore when you select multiple text boxes with text of different colors.

## Testing Instructions



1.) Open up a new story in the Editor.
2.) Click on "T" to open the text panel in the library.
3.) Click on "Heading 1" to add text to editor.
4.) Click on "Paragraph" to add text to editor.
5.) Double click on "dolor" in the paragraph text element.
6.) Change the color of "dolor" to any non-black color (i.e. #FF6600)
7.) It should not crash anymore.

---



#4832
